assembl/views/api2/discussion.py

```diff
@@ -553,13 +560,13 @@ def extract_taxonomy_csv(request):
         else:
             qualify_by_action = "no qualify by action"
         owner_of_the_message = db.query(m.User).filter(m.User.id == query.creator_id).first().name
-        published_on = str(query.creation_date)
+        published_on = str(query.creation_date)[:19]
```

Comment:
put 19 in a const (capitalized) variable and add a comment explaniing how the value of 19 has been obtained